src/aks-preview/HISTORY.rst

```diff
@@ -11,6 +11,7 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 ++++++
+* move loadbalancer/natgateway util functions to azure-cli and update reference in aks-preview project.
```

Comment:
Please update the `azext.minCliCoreVersion` in `azext_metadata.json`

Reply:
bumped azure-cli to 2.49